cogs/memes.py

```diff
@@ -19,9 +19,13 @@
 
 # discord-py requirements
 from discord.ext import commands
+from discord import Embed
 
 # Other utilities
 import random
+from requests import get
```

Comment:
i would probably just do `import requests` and use `requests.get` here to make the global namespace a bit less wierd